ci/build_common.sh

```diff
@@ -127,15 +127,6 @@ function begin_group() {
 #   Group Name: A string specifying the name of the group.
 #   Exit Status (optional): The exit status of the command run within the group. Default is 0.
 function end_group() {
-    local name="${1:-}"
-    local build_status="${2:-0}"
-    local duration="${3:-}"
-    local red="31"
-    local blue="34"
-
-    if [ -n "${GITHUB_ACTIONS:-}" ]; then
-        echo "::endgroup::"
-
```

Comment:
Is that really meant to be removed?

Reply:
No... I have no idea how that even happened.